packages/block-library/src/image/styles.native.scss

```diff
@@ -30,3 +30,26 @@
 	padding-right: 0;
 	padding-bottom: $grid-unit;
 }
+
+.featuredButton {
+	border-top-width: $border-width;
+	border-top-color: $light-gray-400;
+	padding: $grid-unit-20;
+	text-align: center;
+}
+
+.featuredButtonDark {
+	border-top-color: $gray-70;
+}
+
+.setFeaturedButton {
```

Comment:
🤔 I'm a little bit confused by the `featuredButton` vs `setFeaturedButton` distinction. Are there plans for using `featuredButton` in other places and the`setFeaturedButton` style won't be suitable?

If not, what do you think about merging the `featuredButton` and `setFeaturedButton` definitions? In the end, there will be only `setFeaturedButton` and `setFeaturedButtonDark`. I think it won't be a problem that in JS the `color` property will be overwritten for Dark mode and the removeButton case.

Reply:
Thanks for the feedback! My hope was that splitting the selectors out in this way, with two separate selectors for the button's two different styles, would make the code more readable. As it seems to have had the oppositive effect, I've rolled `featuredButton` into `setFeaturedButton`, as per your suggestion. :)